tools/test_utils.py

```diff
@@ -46,9 +47,8 @@ def install_ngraph_bridge(artifacts_dir):
             print("Existing Wheel: " + whl)
         raise Exception("Error getting the ngraph-tf wheel file")
 
-    # First ensure that we have nGraph installed
     ng_whl = os.path.join(artifacts_dir, ngtf_wheel_files[0])
-    command_executor(["pip", "install", "-U", ng_whl])
+    subprocess.call([sys.executable, "-m", "pip", "install", ng_whl])
```

Comment:
TODO: remove once https://github.com/tensorflow/ngraph-bridge/tree/sarkars/current_process_py_to_pip_install is merged